src/components/structures/auth/Registration.js

```diff
@@ -243,7 +243,7 @@ export default createReactClass({
             });
         };
         try {
-            await this._makeRegisterRequest({});
+            await this._makeRegisterRequest(null);
             // This should never succeed since we specified an empty
             // auth object.
```

Comment:
We should update this comment while we're here:
```suggestion
            // This should never succeed since we're doing UIA
```